pcdsdevices/valve.py

```diff
@@ -226,6 +226,22 @@ class VGC(VRC):
     mps_state = Cpt(EpicsSignalRO, ':MPS_FAULT_OK_RBV', kind='omitted',
                     doc=('individual valve MPS state for debugging'))
 
+class VFS(VRC):
+    """Class for Fast Shutter Valve."""
+    vfs_closed = Cpt(EpicsSignalRO, ':CLS_RBV', kind='normal', 
```

Comment:
Is there a reason to prefix them all with 'vfs_'?

Reply:
My thought process was that we are using are already embedding a regular vgc within a vfs, according to the plc code. This is a way to differentiate for now. It may change in the future. Still in the testing phase for fast shutter valve expert screens.

Reply:
There are also these, right?
```
state (:STATE_RBV)
open_limit (:OPN_DI_RBV)
closed_limit (:CLS_DI_RBV)
```

Reply:
Yes, I misread the parent class and missed a layer

Reply:
I'll redo the pull request on Monday. Thanks for your help Tyler and Zach!